clients/src/main/java/org/apache/kafka/clients/producer/internals/TransactionManager.java

```diff
@@ -886,6 +889,9 @@ public void handleResponse(AbstractResponse response) {
             Errors error = addOffsetsToTxnResponse.error();
 
             if (error == Errors.NONE) {
+                log.debug("{}Successfully added partition for consumer group {} to transaction", logPrefix,
+                        builder.consumerGroupId());
```

Comment:
Is it worth including information from `offsets`? Also, is this `partition` or `partitions`? Finally, since this is the `AddOffsetsToTxnHandler`, shouldn't we state something about that? Reading this one quickly, it could be confused with the `AddPartitionsToTxnHandler`.